crates/uv/tests/venv.rs

```diff
@@ -28,7 +26,10 @@ impl VenvTestContext {
         let temp_dir = assert_fs::TempDir::new().unwrap();
         let python_path = python_path_with_versions(&temp_dir, python_versions)
             .expect("Failed to create Python test path");
-        let venv = temp_dir.child(".venv");
+
+        // Canonicalize the virtual environment path for consistent snapshots across platforms
+        let venv = ChildPath::new(temp_dir.canonicalize().unwrap().join(".venv"));
+
```

Comment:
Since we were passing the "absolute" but non-canonicalized path to CLI and we do not canonicalize paths to check if they're relative during user display, the user display differed on macOS.

Reply:
e.g. with debug output in `user_display`

```
path                           = "/var/folders/6p/k5sd5z7j31b31pq4lhn0l8d80000gn/T/.tmpnj31U6/.venv"
CWD.simplified()               = "/private/var/folders/6p/k5sd5z7j31b31pq4lhn0l8d80000gn/T/.tmpnj31U6"
CANONICAL_CWD.simplified()     = "/private/var/folders/6p/k5sd5z7j31b31pq4lhn0l8d80000gn/T/.tmpnj31U6"
path                           = "/var/folders/6p/k5sd5z7j31b31pq4lhn0l8d80000gn/T/.tmpnj31U6/.venv"
```